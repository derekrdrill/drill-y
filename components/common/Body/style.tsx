import styled from 'styled-components';
import { Grid } from '@mui/material';

export const BodyRoot = styled(Grid)({
  zIndex: 0,
});
