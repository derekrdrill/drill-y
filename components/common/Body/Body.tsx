import * as React from 'react';

import { GridProps } from '@mui/material';

import { BodyRoot } from './style';

const Body = ({ children }: GridProps) => (
  <BodyRoot container sx={{ left: 0, position: 'relative', top: { xs: 175, lg: 120 } }}>
    {children}
  </BodyRoot>
);

export default Body;
