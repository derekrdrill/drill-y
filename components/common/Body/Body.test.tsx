import '@testing-library/jest-dom';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';

import Body from './Body';

describe('Body unit tests', () => {
  it('renders as expected', () => {
    const result = renderer
      .create(
        <Body>
          <div>TEST</div>
        </Body>,
      )
      .toJSON();
    expect(result).toMatchSnapshot();
  });
});
