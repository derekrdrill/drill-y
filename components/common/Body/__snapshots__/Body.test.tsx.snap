// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body unit tests renders as expected 1`] = `
<div
  className="MuiGrid-root MuiGrid-container sc-beqWaB fFwJLz css-b6t4rf-MuiGrid-root"
>
  <div>
    TEST
  </div>
</div>
`;
